docs/release_notes/v1.3.0.adoc

```diff
@@ -0,0 +1,78 @@
+[id="{p}-release-notes-v1.3.0"]
+=== Elastic Cloud Control (ecctl) v1.3.0
+++++
+<titleabbrev>v1.3.0</titleabbrev>
+++++
+
+<<{p}-release-notes-v1.3.0-whats-new,What's new>> | <<{p}-release-notes-v1.3.0-bug-fixes,Bug fixes>> | <<{p}-release-notes-v1.3.0-breaking-changes,Breaking changes>> | <<{p}-release-notes-v1.3.0-changelog,Changelog>>
+
+Welcome to the v1.3.0 release of {n}. This version brings new features, some breaking changes, and bug fixes.
+
+Elastic Cloud Control (ecctl) is Elastic’s CLI to manage Elasticsearch Service (ESS) and Elastic Cloud Enterprise (ECE).
+
+Download the release binaries:
+
+[%hardbreaks]
+https://download.elastic.co/downloads/ecctl/1.3.0/ecctl_1.3.0_darwin_amd64.tar.gz[ecctl_1.3.0_darwin_amd64.tar.gz]
+https://download.elastic.co/downloads/ecctl/1.3.0/ecctl_1.3.0_linux_32-bit.deb[ecctl_1.3.0_linux_32-bit.deb]
+https://download.elastic.co/downloads/ecctl/1.3.0/ecctl_1.3.0_linux_32-bit.rpm[ecctl_1.3.0_linux_32-bit.rpm]
+https://download.elastic.co/downloads/ecctl/1.3.0/ecctl_1.3.0_linux_386.tar.g[ecctl_1.3.0_linux_386.tar.g]
+https://download.elastic.co/downloads/ecctl/1.3.0/ecctl_1.3.0_linux_64-bit.deb[ecctl_1.3.0_linux_64-bit.deb]
+https://download.elastic.co/downloads/ecctl/1.3.0/ecctl_1.3.0_linux_64-bit.rpm[ecctl_1.3.0_linux_64-bit.rpm]
+https://download.elastic.co/downloads/ecctl/1.3.0/ecctl_1.3.0_linux_amd64.tar.gz[ecctl_1.3.0_linux_amd64.tar.gz]
+
+[float]
+[id="{p}-release-notes-v1.3.0-whats-new"]
+==== What's new
+
+* *New top level command: `ecctl comment`.* Includes commands to create, delete, list, show and update comments for any of the Elastic Cloud resources: `elasticsearch`, `kibana`, `apm`, `appsearch`, `enterprise_search`, `allocator`, `constructor`, `runner`, `proxy`.
+
+* *New deployment command: `ecctl deployment traffic-filter`.* Includes commands to create, delete, list and show traffic filter rules and the `ecctl deployment traffic-filter association` sub-command includes two more sub-commands to create and delete associations between traffic rules and deployments.
+
+* *New deployment command: `ecctl deployment extension`.* Includes commands to create, delete list and show custom deployment extensions such as Elasticsearch bundles or plugins.
+
+* *Added support for data tiers in all `ecctl deployment` commands.*
+
+[float]
+[id="{p}-release-notes-v1.3.0-bug-fixes"]
+==== Bug fixes
+
+* *Improved deployment tracking.* The deployment tracking has been improved to only fail when the plan has actually finished with a failure, instead of returning errors from in other steps of the plan.
```

Comment:
Not sure if this is any better:

```suggestion
* *Improved deployment tracking.* The deployment tracking has been improved. It now fails only when the plan completes with a failure, instead of returning errors from other steps of the plan.
```

Reply:
Much better! Thanks!